google/cloud/compute/disks/v1/disks_client.cc

```diff
@@ -412,13 +412,14 @@ DisksClient::TestIamPermissions(
 future<StatusOr<google::cloud::cpp::compute::v1::Operation>>
 DisksClient::UpdateDisk(
     std::string const& project, std::string const& zone,
-    std::string const& disk,
+    std::string const& disk, std::string const& update_mask,
     google::cloud::cpp::compute::v1::Disk const& disk_resource, Options opts) {
   internal::OptionsSpan span(internal::MergeOptions(std::move(opts), options_));
   google::cloud::cpp::compute::disks::v1::UpdateDiskRequest request;
   request.set_project(project);
   request.set_zone(zone);
   request.set_disk(disk);
+  request.set_update_mask(update_mask);
```

Comment:
Ignore if this is too complicated...

Should / could we set the update mask ourselves instead of making the customers do it?

i.e. for this method do we know enough to `request.set_update_mask("project,zone,disk")` (or whatever the value is supposed to be).